week-3-practise/level-2/2-map.js

```diff
@@ -0,0 +1,24 @@
+/*
+Only students who have attended enough classes are eligible to sit an exam.
+You have an array named "attendanceCounts"  which contains all the students' names and their attendance counts.
+Produce a new array named "eligibleStudentNames" containing only the NAMES 
+of the students who have attended AT LEAST 8 classes.
+*/
+
+var attendances = [
+  ["Ahmed", 8],
+  ["Clement", 10],
+  ["Elamin", 6],
+  ["Adam", 7],
+  ["Tayoa", 11],
+  ["Nina", 10]
+];
+
+var eligibleStudentNames = attendances.filter(s => s[1] >= 8).map(s => s[0]); // TODO: Complete this line.
```

Comment:
I think this is the answer? We probably want to remove this

Reply:
Thank you  @40thieves for your comments, I have updated my PR and I  removed the answer.